web/src/main/java/se/inera/intyg/webcert/web/service/access/DraftAccessServiceImpl.java

```diff
@@ -190,7 +189,7 @@ public AccessResult allowToForwardDraft(AccessEvaluationParameters accessEvaluat
             .careUnit(accessEvaluationParameters.getUnit())
             .patient(accessEvaluationParameters.getPatient())
             .checkPatientDeceased(true)
-            .invalidCertificateTypeForDeceased(DbModuleEntryPoint.MODULE_ID)
+            .excludeCertificateTypesForDeceased(DbModuleEntryPoint.MODULE_ID)// You should be able to forward a draft for a deceased person
```

Comment:
.invalidCertificateTypeForDeceased() affects every call to this method regardless of AccessEvaluationParameters. This may affect other calls to this method if it's removed. .excludeCertificateTypesForDeceased only affects deceased persons.

Reply:
Based on https://inera.atlassian.net/wiki/spaces/IT/pages/9819521/GE-003+Tillg+ngliga+funktioner the user should not be able to forward a Dödsbevis if the user Is logged in on a different care unit than the draft was created on. 

By adding excludeCertificateTypesForDeceased you will allow the user to do this.